plugin.video.videodevil/resources/settings.xml

```diff
@@ -12,6 +12,7 @@
   <category label="30018">
     <setting id="Ah-Me" type="bool" label="Ah-Me" default="true" />
     <setting id="bitporno.com" type="bool" label="bitporno.com" default="true" />
+	<setting id="CelebsNude" type="bool" label="CelebsNude" default="true" />
```

Comment:
use spaces not tabs  and please also fix the following line

Reply:
Sorry about that, done the change as part of the following commit